applications/autonomous_worlds_on_substrate.md

```diff
@@ -0,0 +1,354 @@
+# SpellCrafter
+###### Autonomous Worlds and Conditional Access Control for Substrate Native Games
+
+- **Team Name:** Ideal Labs and Chainsafe Systems
+- **Funding Details:**
+  - **DOT**: For the **DOT** compensation, please provide a Polkadot address. (ex. multisig, with core contributors)

```

Comment:
We need to define this and I think we're good to go

Reply:
Should we use a multisig?